tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -382,6 +382,17 @@ async def eventual_balance_det(c, wallet_id: str):
             assert list(memos.keys())[0] in [a.name() for a in send_tx_res.spend_bundle.additions()]
             assert list(memos.keys())[1] in [a.name() for a in send_tx_res.spend_bundle.additions()]
 
+            # Test get_transactions to address
+            ph_by_addr = await wallet.get_new_puzzlehash()
+            await client.send_transaction("1", 1, encode_puzzle_hash(ph_by_addr, "xch"))
+            await client.farm_block(encode_puzzle_hash(ph_by_addr, "xch"))
+            await asyncio.sleep(0.5)
```

Comment:
don't sleep in tests, instead use `await time_out_assert(10, wallet_is_synced, True, wallet_node, full_node_api)`, to avoid flaky tests

Reply:
Updated